### PR TITLE
Project import generated by Copybara.

### PR DESCRIPTION
**Pull request by trav3711**
Project import generated by Copybara.

### CODE REVIEW

**Review by charbull — COMMENTED** [review]
can you please run those locally before we merge

**Comment by trav3711** [comment]
Added integration test for Instance validator, need to add integration test for ABEL

**Comment by trav3711** [comment]
Added Integration tests for Instance Validator(no telemetry) and manually tested ABEL.

File: .github/workflows/tools.yml [hunk 1]
```diff
@@ -74,11 +74,16 @@ jobs:
       if: steps.changes.outputs.instance_validator == 'true'
       run: |
         pylint --rcfile ./styles/pylintrc ./tools/validators/instance_validator
-    - name: Instance Yaml Validator Tests
+    - name: Instance Yaml Validator Unit Tests
       if: steps.changes.outputs.instance_validator == 'true' || steps.changes.outputs.type_validator == 'true'
       run: |
        python -m unittest discover -p '*_test.py' --failfast -v
       working-directory: ./tools/validators/instance_validator
+    - name: Instance Yaml Validator Integraion Test
```
Comment (charbull) [inline .github/workflows/tools.yml:82]: interesting edge case for guid, the file used here contains the building guid as `guid123` I think we need to add a guid check to be conform to a uuid4. ok to do this a seperate pr

**Review by charbull — APPROVED** [review]
